src/sync/models/decentralizedOracle.js

```diff
@@ -13,39 +13,37 @@ class DecentralizedOracle {
   }
 
   decode() {
-    this.version = this.rawLog['_version'].toNumber();
-    this.contractAddress = this.rawLog['_contractAddress'];
-    this.eventAddress = this.rawLog['_eventAddress'];
-    this.numOfResults = this.rawLog['_numOfResults'].toNumber();
-    this.lastResultIndex = this.rawLog['_lastResultIndex'].toNumber();
-    this.arbitrationEndBlock = this.rawLog['_arbitrationEndBlock'].toNumber();
-    this.consensusThreshold = this.rawLog['_consensusThreshold'].toJSON();
+    this.version = this.rawLog._version.toNumber();
+    this.contractAddress = this.rawLog._contractAddress;
+    this.eventAddress = this.rawLog._eventAddress;
+    this.numOfResults = this.rawLog._numOfResults.toNumber();
+    this.lastResultIndex = this.rawLog._lastResultIndex.toNumber();
+    this.arbitrationEndBlock = this.rawLog._arbitrationEndBlock.toNumber();
+    this.consensusThreshold = this.rawLog._consensusThreshold.toJSON();
   }
 
   translate() {
-    let optionIdxs = Array.from(Array(this.numOfResults).keys());
-    _.remove(optionIdxs, (num) => {
-      return num === this.lastResultIndex;
-    });
+    const optionIdxs = Array.from(Array(this.numOfResults).keys());
+    _.remove(optionIdxs, num => num === this.lastResultIndex);
 
     return {
       _id: this.contractAddress,
       version: this.version,
       address: this.contractAddress,
       txid: this.txid,
-      topicAddress:this.eventAddress,
+      topicAddress: this.eventAddress,
       status: 'VOTING',
       token: 'BOT',
       name: this.name,
       options: this.options,
-      optionIdxs: optionIdxs,
+      optionIdxs,
```

Comment:
find this syntax confusing

Reply:
same thing, object-shorthand. its a new ES6 standard, but if you want it disabled we can:
https://eslint.org/docs/rules/object-shorthand